### ISSUE TITLE
Monster Power Up Bug (With ScreenShot)

### ISSUE THREAD

**Issue by slyx88** [rathena/rathena#3015]
* **Client Date**: 
2017-06-14

* **Server Mode**: 
Renewal

* **Description of Issue**: 
Monster Power Up skill does not triple the attack power of the monster. Nevertheless the hit bonus works as intended

* Expected Result: 
Melee attack and range attack of the monsters including their skills should do 3 times higher in damage.
* How to Reproduce: 
Summon ifrit and check the damage before and after it uses power up skill.
 * Official Information:
  

* **Modifications that may affect results**: 
none
 
  
![screenrathena006](https://user-images.githubusercontent.com/37906520/38225292-d63a8144-3726-11e8-9db3-4819a25aaae0.jpg)
![screenrathena004](https://user-images.githubusercontent.com/37906520/38225296-df2d831e-3726-11e8-8526-c8c73e771987.jpg)

**Comment by slyx88** [comment]
from status.c it seems the attack of monster is neither watk nor batk. The ATK and ATK2 of the monster are defined as rhw.atk and rhw.atk2

**Comment by Lemongrass3110** [comment]
Could you check if your problem is still there after fe197bf?
This should have fixed a lot of MATK problems with renewal mobs.

**Comment by slyx88** [comment]
The damage increased slightly. I spawned harword bio3 monster. Let him use Maximize Power 1st for damage consistency and check the damage before and after power up. tested on warlock level 175 with 1000 VIT.
Before powerup : 3005
After powerup : 3565

Powerup only affects ATK status and apparently the skill of the mobs are not affected. I managed to get the damage up to official's one by adding this in battle.cpp under //SC effects from caster side.

`if (sc->data[SC_INCATKRATE] ) {
			if (src->type == BL_MOB && !skill_id)	
				damage += damage * sc->data[SC_INCATKRATE]->val1 / 100;	
		}`
